site/src/github/comparison_summary.rs

```diff
@@ -207,7 +207,7 @@ async fn summarize_run(
         "### Overall result: {}{}\n",
         overall_result,
         if is_regression {
-            " - ACTION NEEDED"
+            " - please read the text below"
```

Comment:
I'd keep this in all caps.

Reply:
The reason that I had trouble reengaging with this PR is *precisely this*. This is why. There were a bunch of complaints precisely about the capitals, and you're suggesting putting them back. I have a great deal of trouble communicating my feelings about this sort of thing, I know, but this sort of suggestion is exactly why I have grown to feel the Rust Code of Conduct is a dead letter: it can go on and on about encouraging people to be empathetic, but I don't feel like you're trying to be empathetic with all of the people who feel bothered by this.

Please, do **not**.

Reply:
I just want the message to be clear, and this is an important part of the message. But I'm not wedded to the caps if it's causing problems.

Reply:
I understand the desire to add emphasis, but it ought to be done very carefully for an automated message, and with as little as possible. So for a phrase like this, it might be justified to capitalize a single word (probably PLEASE). This might still come across as off-putting but is more clearly just serving to draw attention.

The first fundamental detail here is that ALL CAPITALS is genuinely *harder* to read. This is because people look for the "shapes" of words. They don't actually read words a letter at a time. Avoiding it unnecessarily is a common recommendation by professional typographers: https://practicaltypography.com/all-caps.html

I can pull studies on that if you want more citations, but making something harder to read is more likely to cause people's eyes to glaze over and ignore it.

Reply:
and since we're talking about programmers that we are asking to do additional work, if their eyes glaze over and they disengage, that seems likely to either cost us the contribution (they won't be interested in satisfying the perfbot yelling at them) which may be well worth the "regression".

or if it's an already-merged PR, they simply won't follow up.
or they may follow up but be anxious about it, making it less likely they make further contributions.

Reply:
@workingjubilee Could you please confirm if the current version looks good to you?